src/app/content/components/TableOfContents/styled/wrapper.tsx

```diff
@@ -97,35 +106,50 @@ export const SidebarBody = styled.div<{isOpen: State['tocOpen']}>`
 export const ToCHeader = styled.div`
   display: flex;
   align-items: center;
-  height: ${toolbarDesktopHeight}rem;
+  justify-content: space-between;
+  height: ${topbarDesktopHeight}rem;
+  padding: 0 ${sidebarPadding}rem;
   overflow: visible;
+  border-bottom: 1px solid ${theme.color.neutral.formBorder};
   box-shadow: 0 1rem 1rem -1rem rgba(0, 0, 0, 0.14);
   ${theme.breakpoints.mobile(css`
-    height: ${toolbarMobileHeight}rem;
+    height: ${topbarMobileHeight}rem;
   `)}
 `;
 
-// tslint:disable-next-line:variable-name
-export const TimesIcon = styled((props) => <Times {...props} aria-hidden='true' focusable='false' />)`
-  ${toolbarIconStyles};
-  margin-right: 0;
-  padding-right: 0;
-  color: ${toolbarIconColor.lighter};
+// tslint:disable-next-line: variable-name
+export const ToCHeaderText = styled.span`
+  font-size: 1.8rem;
+  line-height: 2.9rem;
+  font-weight: 600;
+  margin: 0;
+  padding: 0;
+  color: ${toolbarIconColor.base};
+  flex: 1 1 0%;
+  text-align: left;
+  ${theme.breakpoints.mobileMedium(css`
+    text-align: center;
+  `)}
+`;
+
+// tslint:disable-next-line: variable-name
+export const CloseToCAndMobileMenuButton = styled((props) => {
```

Comment:
We need to be able to close the TOC without closing the mobile menu because that's what the back arrow is for. But I think it's ok to close the TOC whenever we close the mobile menu. In that case I think I could just move these styles over to SidebarControl and rename some things.